.changeset/chatty-planets-trade.md

```diff
@@ -0,0 +1,18 @@
+---
+'@celo/contractkit': major
+'@celo/celocli': major
+'@celo/connect': patch
+'@celo/utils': patch
+'@celo/base': patch
+---
+
+The following changes are related to adding support for more fee currencies in the @celo packages.
```

Comment:
I would have one changeset file per package since this way the change log for all changed packages will all have changest for other packages